test/TestRole.sol

```diff
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.0;
 
-import "test/TestSetup.sol";
+import "test/TestTransactionSetup.sol";
 
-contract TestRole is TestSetup {
```

Comment:
Are these tests supposed to inherit `TestTransactionSetup`? Shouldn't it only be tests under `test/transactions/`?

Reply:
I depend of what we want to do if we want to test the impact of the transaction on the Dao setup it could be good this way but if we do not want, I can revert the changes.

Reply:
What do you think @Rubilmax, @MerlinEgalite @pakim249CAL, @makcandrov  ?

Reply:
I didn't think about it but you're right, better this way!